website/activemembers/api/v2/views.py

```diff
@@ -26,6 +23,12 @@ class MemberGroupListView(ListAPIView):
 
     serializer_class = MemberGroupListSerializer
     queryset = MemberGroup.active_objects.all()
```

Comment:
```suggestion
    queryset = MemberGroup.objects.all()
```
This way the you can see old committees as well. If you want to filter on 'active' ones, there's the date filter.

Reply:
@se-bastiaan You requested to change it to `active_objects` so could you maybe comment on this?

Reply:
I am not sure we should be exposing non-active committees outside of the admin, it's really more a published/non-published situation where people who are not admins should not have access to those objects

Reply:
Boards are groups that are [always active](https://github.com/svthalia/concrexit/blob/master/website/activemembers/models.py#L164) since they contain extra constraints like that there can only be one board within a certain time period.

So yes, for other types of groups this really means that they should not be shown anymore.